.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,25 @@
+name: Bump Version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+jobs:
+  bumpVersion:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Bump Version
+        run: |
+          sed -i s/QVersionNumber::fromString\(.*\)/QVersionNumber::fromString\(\"${{ github.event.inputs.version }}\"\)/ src/window.cpp
```

Comment:
lets put this in a Make file, so when we update the logic of updating the version, it would be part of the general Make 

"make bump-version"

Reply:
Moved to Makefile